ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -33,7 +32,7 @@ import {
   handleSettingsRefs,
 } from '../../../helpers/utils/settings-search';
 
-const CORRUPT_JSON_FILE = 'CORRUPT_JSON_FILE';
+// const CORRUPT_JSON_FILE = 'CORRUPT_JSON_FILE';
```

Comment:
Could we delete all of this commented code? We can easily restore these changes from the git history if we need them again, there is no benefit to keeping around commented code

Reply:
Good suggestion, removed all legacy code now 🙏🏻